apps/publisher/controllers/apis-router.jag

```diff
@@ -34,6 +35,7 @@ var mapper = function(path) {
         return path;
     };
 };
+metrics.start("apis-router","router");
```

Comment:
Thanks @lahirus , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/1248edce9638d747e0ed29a9b4794167b4ceab0d)